content/communities/manage-listserv-subscription.md

```diff
@@ -40,6 +40,18 @@ When sending a message to the entire Listserv, consider that your message is goi
 - Stay organized: Please keep the conversation threads focused. Start a new thread if it does not pertain to the latest conversation.
 - Be brief: Try to be direct and stay on point as much as possible.
 
+### Other considerations
```

Comment:
I suggest making this an H2 instead of an H3.

Reply:
Just made an update!